docs/application/dotnet/guides/user-interface/nui/relative-layout.md

```diff
@@ -73,14 +82,533 @@ The `HorizontalAlignment` and the `VerticalAlignment` properties describe how to
+```
+
+</span>
+                </tr>
+            </tbody>
+        </table>
+    </div>
+</div>
+
+### Green view
+
+The following code shows how create greenView to relate to yellowView for every target.
+
+![RelativeLayoutExampleGreenView](./media/relative_layout_example_green_view.png)
+
+<div id="TabSection5">
+    <div class="sampletab " id="ProjectCreateTab">
+        <button id="RelativeLayout-GreenView-CSharp" class="tablinks " onclick="openTabSection(event, 'RelativeLayout-GreenView-CSharp', 'TabSection5') ">C#</button>
+        <button id="RelativeLayout-GreenView-Xaml" class="tablinks " onclick="openTabSection(event, 'RelativeLayout-GreenView-Xaml', 'TabSection5') ">Xaml</button>
+    </div>
+    <div id="RelativeLayout-GreenView-CSharp" class="tabcontent">
+        <table>
+            <tbody>
+                <tr>
+<span style="display:block">
+
+- View's targets are set to the `yellowView`.
+- `RelativeLayout.SetLeftRelativeOffset(greenView, 0.0f)` - view's left is at the yellowView's left.
+- `RelativeLayout.SetRightRelativeOffset(greenView, 0.2f)` - view's right is at the position of 0.2 width from the yellowView's left.
+- `RelativeLayout.SetTopRelativeOffset(greenView, 0.0f)` - view's top is at the yellowView's top.
+- `RelativeLayout.SetBottomRelativeOffset(greenView, 0.2f)` - view's bottom is at the position of 0.2 width from the yellowView's left.
+- `RelativeLayout.SetFillHorizontal=(greenView, true)` and `SetRelativeLayout.FillVertical(greenView, true)` - Since the view does not have its Width and Height,the view should be filled the relative position(left, right, top, bottom).
+
+```csharp
+View greenView = new View
+{
+    BackgroundColor = Color.Green
+};
+
+RelativeLayout.SetLeftTarget(greenView, yellowView);
+RelativeLayout.SetRightTarget(greenView, yellowView);
+RelativeLayout.SetTopTarget(greenView, yellowView);
+RelativeLayout.SetBottomTarget(greenView, yellowView);
+
+RelativeLayout.SetLeftRelativeOffset(greenView, 0.0f);
+RelativeLayout.SetRightRelativeOffset(greenView, 0.2f);
+RelativeLayout.SetTopRelativeOffset(greenView, 0.0f);
+RelativeLayout.SetBottomRelativeOffset(greenView, 0.2f);
+
+RelativeLayout.SetFillHorizontal(greenView, true);
+RelativeLayout.SetFillVertical(greenView, true);
+relativeLayoutView.Add(greenView);
+```
+
+- View's targets are set to the `yellowView`.
+- `RelativeLayout.LeftRelativeOffset="0.0"` - view's left is at the yellowView's left.
+- `RelativeLayout.RightRelativeOffset="0.2"` - view's right is at the position of 0.2 width from the yellowView's left.
+- `RelativeLayout.TopRelativeOffset="0.0"` - view's top is at the yellowView's top.
+- `RelativeLayout.BottomRelativeOffset="0.2"` - view's bottom is at the position of 0.2 width from the yellowView's left.
+- `RelativeLayout.FillHorizontal="true"` and `RelativeLayout.FillVertical="true"` - Since the view does not have its Width and Height,the view should be filled the relative position(left, right, top, bottom).
```

Comment:
```suggestion
- `RelativeLayout.FillHorizontal="true"` and `RelativeLayout.FillVertical="true"` - Since the view does not have its Width and Height, the view should be filled the relative position(left, right, top, bottom).
```